src/main/java/org/sagebionetworks/web/client/view/NewAccountViewImpl.java

```diff
@@ -84,23 +84,31 @@ public void init() {
 		password2Field.addKeyDownHandler(register);
 
 		userNameField.addBlurHandler(event -> {
-			checkUsernameFormat();
+			boolean enableNext = checkUsernameFormat();
+			reconfigurePageProgressWidget(enableNext);

```

Comment:
And I noticed that the page progress widget was not being rendered on the NewAccount page (after clicking on the email link).  So reconfigure it whenever something in the form changes (so Next is usually not available until the errors have been dealt with).